### DISCUSSION TITLE
Updating the Internet Archive mirror?

### DISCUSSION THREAD

**Discussion by JamesR98** [freelawproject/courtlistener#4798] [General]
It seems to me RECAP has not mirrored any files since Oct 9 when archive.org stopped accepting uploads to make changes to security, archive.org now accepts uploads again. I assume Free Law is well aware of but any timeline as to when they might be backed? I used the Internet Archive's api to handle cases so it would be useful to me and I assume a few others. Thanks for all your invaluable work!

**Comment by mlissner** [comment]
What makes you think we're not uploading? Our latest cronjob says: 

```
INFO Uploading free opinions to Internet Archive.                                                                                                                                                                                             
INFO Sending 1382 items to Internet Archive.                                                                                                                                                                                                  
INFO Sent 1000/1382 tasks to celery so far.                                                                                                                                                                                                   
INFO Uploading non-free PDFs to Internet Archive.                                                                                                                                                                                             
INFO Sending 4343 items to Internet Archive.                                                                                                                                                                                                  
INFO Sent 1000/4343 tasks to celery so far.                                                                                                                                                                                                   
INFO Sent 2000/4343 tasks to celery so far.                                                                                                                                                                                                   
INFO Sent 3000/4343 tasks to celery so far.                                                                                                                                                                                                   
INFO Sent 4000/4343 tasks to celery so far.                                                                                                                                                                                                   
INFO Uploading oral arguments to Internet Archive.                                                                                                                                                                                            
INFO Sending 94 oral argument files to Internet Archive 
```

One thing that might be relevant is that we have been doing a lot of data gathering, and had a few days where we uploaded tens of thousands of documents.

**Comment by JamesR98** [comment]
If you see recapinfo it's last archived item was on Oct 9:
https://archive.org/details/@recapinfo?sort=-publicdate
As well a few of the court cases I follow have not had anything uploaded since that day either. I take it then courtlistener is just working through it's backlog, which is what I was hoping to hear.

**Reply by mlissner** [reply]
Yeah, that's all pretty weird. We're uploading just like we always have, I think. It sort of seems like AI is accepting our uploads, but failing to update the HTML pages to show people the uploads exist. Could it be that their systems are still quite broken?

**Reply by JamesR98** [reply]
Other people's uploads seem to be just fine:
https://archive.org/search?query=court&sort=-publicdate

I think after the hack IA had people change their passwords, you may need to change yours and then run `ia configure`. I would  try checking if those pdfs are actually being uploaded. If not reaching out to IA seems to be the next step, I'm sure they're a lot more interested in your account than a lot of the other accounts they get.

**Reply by mlissner** [reply]
Weird. I just reached out to them to see if they have any advice. Thanks for raising this.

**Reply by mlissner** [reply]
Looks like we had a handful of issues here (but it's mostly IAs' fault). Fixes being developed here: https://github.com/freelawproject/courtlistener/issues/4812. Expect uploads tonight.
